store/auth.ts

```diff
@@ -27,7 +27,8 @@ const initState = (): AuthState => {
 }
 
 export const useAuthStore = defineStore('auth', () => {
-    const { proxy: root } = getCurrentInstance() || {}
+    const store = useAuthStore()
```

Comment:
the store calls itself?

Reply:
yea we can access the context of the store and access nuxt context

Reply:
Would this be available for all stores? i.e. accessing the multi api